src/lighteval/metrics/llm_as_judge.py

```diff
@@ -89,40 +90,65 @@ def __init__(
         # the second is for the backup case: [score]
         self.one_score_pattern = re.compile(r"\[\[(\d+\.?\d*)\]\]")
         self.one_score_pattern_backup = re.compile(r"\[(\d+\.?\d*)\]")
```

Comment:
We might want to provide these as kwargs instead

Reply:
mhh, I think it's cleaner to have them in the class, we can do another PR to add to have better prompt and overall definition for the judges